sites/public/pages/eligibility/age.scss

```diff
@@ -0,0 +1,7 @@
+.age-field input {
+  max-width: 3.5rem;
+}
+
+.age-field label {
+  margin-left: .5rem;
```

Comment:
ditto here. it'd be the `ml-2` class to apply here I think

Reply:
I moved this styling into ui-components like we talked about.